src/Uno.UI.Tests/Helpers/AssertEx.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Windows.UI;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+
+namespace Uno.UI.Tests.Helpers
+{
+	public static class AssertEx
+	{
+		public static void AssertContainsColorBrushResource(ResourceDictionary resources, string key, Color expected)
+		{
+			Assert.IsNotNull(resources);
+
+			AssertHasColor(resources[key] as Brush, expected);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'resources' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=4826539)